README.md

```diff
@@ -114,7 +114,7 @@ var_dump(ast\parse_code($code, $version=70));
 // Output:
 object(ast\Node)#1 (4) {
   ["kind"]=>
-  int(133)
```

Comment:
This matches the value in php 8.0 and php 7.4.

Note that the value of the ast node kinds varies based on what PHP's minor version is - For example, in php 7.0 when this was initially written, this actually was 133.

Probably makes sense to change to 132, though

Reply:
Thanks for the comments, it makes sense. I ran the code against php 7.4 and got 132. And since AST_STMT_LIST = 132 in ast_stub.php as well, I thought we'd better change it.